packages/launch/src/focused-launch/summary/index.tsx

```diff
@@ -130,8 +130,10 @@ const DomainStep: React.FunctionComponent< DomainStepProps > = ( {
 							>
 								{ info }
 							</Tooltip>
-							{ /* @TODO: should use <Subtitle/> once https://github.com/Automattic/wp-calypso/issues/47418 is solved */ }
-							<p className="focused-launch-summary__mobile-commentary focused-launch-summary__mobile-only">
+							<SubTitle
+								tagName="p"
+								className="focused-launch-summary__mobile-commentary focused-launch-summary__mobile-only"
+							>
```

Comment:
This `<p>` tag actually didn't need to be converted to `<Subtitle />` — sorry, my bad!
Please revert the change (but still delete the TODO comment)

Reply:
I just reverted the changes and left the comment out.